opentelemetry/src/lib.rs

```diff
@@ -88,6 +88,17 @@
 //! See the [`metrics`] module docs for more information on creating and
 //! managing instruments.
 //!
+//!
+//! //! # Logs
+//!
+//!  The [`logs`] module contains the Logs Bridge API. It is not intended to be
+//!  called by application developers directly. It is provided for logging
+//!  library authors to build log appenders, that bridges existing logging
+//!  systems with OpenTelemetry. OpenTelemetry. Bridges for
```

Comment:
```suggestion
//!  systems with OpenTelemetry. Bridges for
```
Hmmmm why do we have the second `Opentelemetry.` here

Reply:
Fixed!